2018-10-Vancouver/agenda.md

```diff
@@ -64,31 +64,31 @@ _We are expecting 70-100 people to participate_.
-| 9:00          | Main (Room 121)        | Intro, Code of Conduct                                              | @mcollina @dshaw @chowdhurian |                                            |
-| 9:15 - 10:45  | Main (Room 121)        | Modules                                                             | @MylesBorins                  | https://github.com/nodejs/modules/issues/177 |
-| 9:15 - 10:45  | Breakout 1 (Room 118)   | Website Redesign                                                    | @amiller-gh                   | https://github.com/nodejs/website-redesign/issues/88 |
-| 9:15 - 10:45  | Breakout 2 (Room 119)   | Security WG                                      |        @mhdawson         | https://github.com/nodejs/security-wg/issues/358 |
-| 10:45         |              | __Coffee Break__                                                    |                               |  |
-| 11:00 - 12:30 | Main (Room 121)        | Promises                                      | @BridgeAR                              | |
-| 11:00 - 12:30 | Breakout 1 (Room 118)        | Governance                                                          |  @amiller-gh                  | https://github.com/nodejs/admin/issues/228  |
+| Time          | Place        | Topic                                                               | Speakers/Champions            | Issue                                      | Slides                                    |
+|---------------|--------------|---------------------------------------------------------------------|-------------------------------|--------------------------------------------|-------------------------------------------|
+| 8:00          |              | Breakfast and registration                                          |                               |                                            | |
+| 9:00          | Main (Room 121)        | Intro, Code of Conduct                                              | @mcollina @dshaw @chowdhurian |                                            | |
+| 9:15 - 10:45  | Main (Room 121)        | Modules                                                             | @MylesBorins                  | https://github.com/nodejs/modules/issues/177 | |
+| 9:15 - 10:45  | Breakout 1 (Room 118)   | Website Redesign                                                    | @amiller-gh                   | https://github.com/nodejs/website-redesign/issues/88 | |
+| 9:15 - 10:45  | Breakout 2 (Room 119)   | Security WG                                      |        @mhdawson         | https://github.com/nodejs/security-wg/issues/358 | |
+| 10:45         |              | __Coffee Break__                                                    |                               |  | |
+| 11:00 - 12:30 | Main (Room 121)        | Promises                                      | @BridgeAR                              | | [Code examples](https://gist.github.com/mmarchini/1f7476e0ee21a1f17832971289c3f4b5)|
```

Comment:
This gist was presented in the Diagnostics session. Can you move it there?

Reply:
Yeah! I’ll do today :D